mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/DefaultImplementingTypeAnnotation.java

```diff
@@ -44,6 +44,11 @@ public String getName() {
   }
 
   @Override
+  public boolean isPublic() {
+    return false;
```

Comment:
sure this isn't required by studio/mozart for Nullsafe(defaultingTo)?

Reply:
this never worked. If you look closely at the metadata PR, you'll see that I fixed a bug in which if the deserealization of a TypeAnnotation fails, it swallows the exception and continues. once I fixed that, this started to fail because the TypeANnotationSerializer doesn't know how to serialize a metadata type.

Also, if Studio is using this, they should no longer do it through the json. So, I think our best play here is to make this private and revisit IF Studio has a problem with this